### PR TITLE
make: strict option

### LINKED ISSUE #410: Some server issue
After upgrade to `enb^1.0.1` (https://github.com/bem/project-stub/commit/7b3b66f4a36819bf90c782fc8fcfa2ee5de790c5) there are a few `build failed` logs in console on each page refresh in `project-stub` (still everything works fine and no such logs appear on `enb make`).

### LINKED ISSUE #411: make: strict option
We need to add the strict option to make command. 

If strict: true then as now assembly will fails with TargetNotFoundError. If strict: false then ENB will ignore such targets. 

By default — strict: true.

It need for ENB server. See #410.

### PR DESCRIPTION
**Pull request by blond**
Resolved #411 and fixes #410.

### CODE REVIEW

**Comment by tadatuta** [comment]
lgtm

File: lib/api/make.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 var path = require('path'),
     vow = require('vow'),
+    colors = require('../ui/colorize'),
     MakePlatform = require('../make'),
-    cdir = process.cwd();
+    cdir = process.cwd(),
+    TargetNotFoundError = require('../errors/target-not-found-error');
```
Comment (shuhrat) [inline lib/api/make.js]: IMHO strange name =)
I prefer something like this `../errors/target-not-found`

File: lib/api/make.js [hunk 2]
```diff
@@ -54,6 +63,8 @@ module.exports = function (targets, options) {
                         console.log(graph.render());
                     }
 
+                    logger.log('build finished - ' + colors.red((new Date() - startTime) + 'ms'));
```
Comment (shuhrat) [inline lib/api/make.js]: format('build finished - %s ms',  colors.red((new Date() - startTime))

**Pull request merged**